build.sbt

```diff
@@ -56,14 +54,14 @@ lazy val corJourney = Project(appName + "-cor-journey", file("cor-journey"))
  * Collection Of Routines - test data
  */
 lazy val corJourneyTestData = Project(appName + "-cor-journey-test-data", file("cor-journey-test-data"))
-  .enablePlugins(SbtAutoBuildPlugin, SbtGitVersioning)
-  .settings(commonSettings: _*)
+  .settings(commonSettings *)
   .settings(
     scalaVersion := scalaV,
     majorVersion := majorVer,
     libraryDependencies ++= List(
-      "org.playframework" %% "play"      % play.core.PlayVersion.current % Provided,
-      "org.playframework" %% "play-test" % play.core.PlayVersion.current % Provided
+      "uk.gov.hmrc"       %% "bootstrap-common-play-30" % AppDependencies.bootstrapVersion % Provided,
```

Comment:
after removing auth-client-play-30 from corJourney, needed to add this that will provide http client for tests that is already a dependency in bootstrap-common